### ISSUE TITLE
Update accessibility statement to say that mobile tabs are not in a `tablist`

### ISSUE THREAD

**Issue by EoinShaughnessy** [alphagov/govuk-design-system#1654]
## What
Update our [accessibility statement](https://design-system.service.gov.uk/accessibility/) to explain a mobile tabs issue that fails WCAG 2.1.

## Why
As explained in [#1653: Code block tabs on mobile are not contained within a `tablist`](https://github.com/alphagov/govuk-design-system/issues/1653):

The HTML and Nunjucks 'tabs,' when run on the 'mobile' layout of the Design System, are not contained within a `tablist`.

Tabs on mobile are different to tabs you see on tablet and other devices. On mobile, the tabs act more like an accordion, and the elements need to be in a different position in the DOM.

The tablet / desktop tabs are correctly nested within a `ul role="tablist"`:

https://github.com/alphagov/govuk-design-system/blob/1fe2b366c1d13c457e6a210aaf54c3e7bc106774/views/partials/_example.njk#L54-L58

However, the mobile tabs are not:

https://github.com/alphagov/govuk-design-system/blob/1fe2b366c1d13c457e6a210aaf54c3e7bc106774/views/partials/_example.njk#L77

This means that the tabs are [not contained by their required parent element](https://web.dev/aria-required-parent/) – as [ARIA 1.1 says](https://www.w3.org/TR/wai-aria-1.1/#tab):

> Authors MUST ensure elements with role `tab` are contained in, or owned by, an element with the role `tablist`.

This fails [WCAG 2.1 1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html).

## Who needs to know about this
Developers, Technical Writer

## Done when
- [x] Tech writer drafts content update
- [x] Content update passes dev review
- [x] Content update passes 2i
